src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -234,13 +226,11 @@ export default function TokenDetails({
 
         <RightPanel onClick={() => isBlockedToken && setOpenTokenSafetyModal(true)}>
           <div style={{ pointerEvents: isBlockedToken ? 'none' : 'auto' }}>
-            <Widget
-              defaultTokens={{
-                [Field.INPUT]: inputToken ?? undefined,
-                default: detailedToken ?? undefined,
+            <Swap
+              prefilledState={{
```

Comment:
note that the `inputCurrency` default is handled automatically by the default initialization of the SwapState reducer - it checks URL params for `inputCurrency` and `outputCurrency` on the main page, and we're just re-using the same variable name on TDP